src/content/blog/announcements/2024-11-30_nixos-24.11.md

```diff
@@ -0,0 +1,52 @@
+---
+id: nixos-24.11
+title: NixOS 24.11 released
+date: 2024-11-30T11:00:00.000Z
+category: announcements
+---
+
+[![NixOS 24.11 Vicuña logo](../../../assets/logo/nixos-logo-24.11-vicuna-lores.png)](https://github.com/NixOS/nixos-artwork/blob/master/releases/24.05-vicuna/vicuna.png)
```

Comment:
`24.11-vicuna` in the path to `nixos-artwork` needs to be changed.

Reply:
I fixed the path.